framework/tests/src/tests/AnnotatedForTest.java

```diff
@@ -17,7 +17,7 @@ public AnnotatedForTest(File testFile) {
               "subtyping",
               "-Anomsgtext",
               "-Aquals=tests.util.SubQual,tests.util.SuperQual",
-              "-AuseSafeDefaultsForUnannotatedSourceCode");
+              "-AuseDefaultsForUncheckedCode=source,bytecode");
```

Comment:
I added the only test that is currently under conservative-defaults/annotatedfor. I recommend making this "source,-bytecode" for now. That way anyone who looks at this knows exactly what is being tested, and if they choose to add new tests to the directory or move the test to another directory, they'll know what to do.